### ISSUE TITLE
[ManagerManager] Moving multiple URL TV's to new tab ends in poor layout.

### ISSUE THREAD

**Issue by BBloke** [evolution-cms/evolution#295]
I've dug into ManagerManager and found it brilliant.  I decided to move some TVs to a tab of their own.  The problem is that my TVs are URLs and they don't format as expected on the page.

my ManagerManager Rules:
```
mm_createTab('MOVIE', 'movie', '', '12', '', '');
mm_createTab('MOVIE LINKS', 'movielinks', '', '12', '', '');
mm_moveFieldsToTab('MovieCategory,SpecialMovieTag,MovieAwards,WebUserOwner,Userid,MovieCreator,MovieLength,MovieProductionDate,MovieGameEngine,MovieEmail','movie', '', '12');
mm_moveFieldsToTab('MovieURL,MovieDownload,MovieProducerProfile,MovieImage,MiddleSideBar', 'movielinks', '', '12');
```
Here's a standard web resource page section of URL TVs
![tvurls_standard](https://user-images.githubusercontent.com/24464015/31670036-a6a421b2-b34e-11e7-9c54-93b1d090f8d4.jpg)

Here's what it looks like on a new tab:
![tvurls_newtab](https://user-images.githubusercontent.com/24464015/31670062-b5de461c-b34e-11e7-9dca-eb50ca786fa3.jpg)

**Comment by fourroses666** [comment]
Yea love mm_rules too.
Could it be a style (floating) issue?

**Comment by Nicola1971** [comment]
evo version? 
i've the same issue with a site with evo 1.3.6
![download](https://user-images.githubusercontent.com/7342798/31677563-768d2736-b36b-11e7-96c2-8bf55d427b9a.png)

**Comment by BBloke** [comment]
Could well be CSS I guess. I will dig a little deeper.  I had all the TV's on one line in one tab page and thought it would be good to split it to two tabs.  Same thing!

**Comment by BBloke** [comment]
I dug a little deeper and it's not CSS.  It appears to be a formatting issue relating to the generation of tables when producing the TV.

On a standard page there is a table for the whole section.
Each entry starts a new row.
The description is the first cell and the details are the second cell.
The second cell is a new table which contains the details of the TV.
This new table does not get completed properly but continues to render the next TV.
From the second TV onwards the information is rendered correctly.

I'm not sure whether it's an MM fault when moving the TVs or not.

**Comment by BBloke** [comment]
evo 1.3.6

**Comment by bossloper** [comment]
From what I can tell when a tv is moved to another tab the containing table no longer has 100% width set.

source html for a tv that has NOT been moved by ManagerManager:
```
<table border="0" cellspacing="0" cellpadding="0" width="100%">
<tbody>
<tr class="urltv">
<td>
<select id="tv9_prefix" name="tv9_prefix" onchange="documentDirty=true;">
etc...
```

source html for a tv that has HAS been moved by ManagerManager to a new tab:
```
<table border="0" cellspacing="0" cellpadding="0">
<tbody>
<tr class="urltv">
<td>
<select id="tv9_prefix" name="tv9_prefix" onchange="documentDirty=true;">
etc...
```

Don't know how to fix it though!

**Comment by BBloke** [comment]
I don't think it's the width, it's the html. It's not finishing the first TV correctly so the 2nd TV starts within the first TVs nested table (effectively the 2nd cell).

The 2nd TV onwards has the correct html formatting.

That what I could see in chrome inspect.

**Comment by fourroses666** [comment]
Maybe a closing </tr> bug when using dropdown TVs? Just guessing

**Comment by dmi3yy** [comment]
fixed in 2.0-alpha
